src/api-service/__app__/onefuzzlib/azure/subnet.py

```diff
@@ -10,15 +10,16 @@
 from azure.core.exceptions import ResourceNotFoundError
 from msrestazure.azure_exceptions import CloudError
 from onefuzztypes.enums import ErrorCode
-from onefuzztypes.models import Error
+from onefuzztypes.models import Error, NetworkConfig
+from onefuzztypes.primitives import Region
 
 from .network_mgmt_client import get_network_client
 
 
-def get_subnet_id(resource_group: str, name: str) -> Optional[str]:
+def get_subnet_id(resource_group: str, name: str, subnet_name: str) -> Optional[str]:
```

Comment:
What's the difference between `name` and `subnet_name`?

Reply:
right now, nothing.  we have only one subnet name.